src/pages/User/components/PullRequests/UserInfo/Message/getMessage.js

```diff
@@ -0,0 +1,44 @@
+import { pullRequestAmount, otherReposAmount } from '../../pullRequestAmount';
+
+const messages = [
+  "It's not too late to start!",
+  'Off to a great start, keep going!',
+  'Keep going!',
+  'Keep it up!',
+  'Nice work, Keep it up !',
+  "Nice! Now, don't stop!",
+  "Keep Going, don't stop!",
+  'So close!',
+  'So close! Keep it up !',
+  'Way to go!',
+  "Now you're just showing off!"
+];
+
+/**
+ * Get Messages Based on Pull Request and Other Repo count.
+ *
+ * @param {*} pullRequestCount
+ * @param {*} otherReposCount
+ * @returns {string}
+ */
+function getMessage(pullRequestCount, otherReposCount) {
+  const currentMonth = new Date().getMonth();
+
+  if (currentMonth < 9) {
+    return "Last year's result.";
+  }
+
+  if (currentMonth > 9) {
+    return "This year's result.";
+  }
+
+  const isShowingOff = pullRequestCount > pullRequestAmount && otherReposCount > otherReposAmount;
+
+  if (isShowingOff) {
+    return messages[messages.length - 1];
```

Comment:
return the text directly here.

```js
return "Now you're just showing off!";
```

Reply:
will it be good to write the string as proposed directly, if change is needed we would have to change at two places.

Reply:
Makes sense.